### DISCUSSION TITLE
MCU boot for STM32H743

### DISCUSSION THREAD

**Discussion by macharlachanakya** [mcu-tools/mcuboot#2009] [General]
Hello Techies, 

We are using STM32H743, we would like to run mcuboot on stm32h743, but we haven't find the example for stm32h7xx devices, 

Is there any specific reason not to provide mcuboot example for stm32h7 devices?

**Comment by dillonj-sentient** [comment]
There are so many targets that MCUboot can be ported to and I guess no one has provided a generic example for this microcontroller family. I am using the STM32U5 for which there is also no example, so I followed along the porting guide at https://interrupt.memfault.com/blog/mcuboot-overview and was able to have success porting MCUboot to my board.

**Reply by tech1023** [reply]
One thing to note though is that the STM32U5 is a new generation with an M33 not M7 core and so the device supports trustzone so is probably easier to port than the STM32H7
